security/openshift-security/scc.yaml

```diff
@@ -18,7 +18,7 @@ requiredDropCapabilities:
 runAsUser:
    # Here, the allowed user ID is configured to be between 1001 and 1005
   type: MustRunAsRange
-  uidRangeMin: 1001
```

Comment:
+1 we also have to update here https://github.com/confluentinc/confluent-kubernetes-examples/tree/master/security/openshift-security#advanced-install-confluent-for-kubernetes-with-custom-scc-policy

Reply:
Added that as well